custom_components/xiaomi_miio_airpurifier/fan.py

```diff
@@ -2013,15 +2014,15 @@ async def async_set_display_off(self):
             False,
         )
 
-    async def async_set_display_orientation(self, orientation: str):
+    async def async_set_display_orientation(self, display_orientation: str):
         """Set the display orientation."""
         if self._device_features & FEATURE_SET_DISPLAY_ORIENTATION == 0:
             return
 
         await self._try_command(
             "Setting the display orientation of the miio device failed.",
             self._device.set_display_orientation,
-            AirfreshT2017DisplayOrientation[orientation.title()],
+            AirfreshT2017DisplayOrientation[display_orientation],
```

Comment:
Could you use `display_orientation.title()` here?

Reply:
No, it makes `LandscapeLeft` to `Landscapeleft`, which doesn't work

Reply:
Ups!